frontend/src/containers/SignInPage/index.tsx

```diff
@@ -0,0 +1,67 @@
+import * as React from 'react';
+import {useForm} from 'react-hook-form';
+import { yupResolver } from '@hookform/resolvers/yup';
+import * as yup from 'yup';
+// import { isValidEmail, isValidPassword } from 'healthcare-shared/helpers';
+
+import styles from './signin.module.scss';
+
+enum RegisterPayloadKey {
+  EMAIL = 'email',
+  PASSWORD = 'password',
+}
+interface IRegisterPayload {
+  [RegisterPayloadKey.EMAIL]: string;
+  [RegisterPayloadKey.PASSWORD]: string;
+}
+/* const DEFAULT_VALUES: IRegisterPayload = {
+  [RegisterPayloadKey.EMAIL]: '',
+  [RegisterPayloadKey.PASSWORD]: '',
+}; */
+
+const SignInPage: React.FC = () => {
+  const validationSchema = yup.object().shape({
+      email: yup.string().email('Email is incorrect').required('Email is required'),
+      password: yup.string().length(6, 'min 6 chars').required('Password is required min six symbols')
```

Comment:
See my examples with validation schemas. We need to add more constants for `magic numbers` and error messages